src/world/partners.c

```diff
@@ -55,7 +55,7 @@ s32 D_800F7FF4 = 4;
 s32 D_800F7FF8 = 5;
 s32 D_800F7FFC = 7;
 s32 D_800F8000[] = { 8, 0, 0, 0 };
-s32 D_800F8010[] = { 0x003251D0, 0x00325AD0, (s32)&D_802C05CC, 0x00000000 };
+s32 D_800F8010[] = { &code_code_3251D0_ROM_START, &code_code_3251D0_ROM_END, (s32)&D_802C05CC, 0x00000000 };
```

Comment:
Give this a struct and use field names in `use_consumable`

Reply:
I feel like that's more documentation-y and seems not necessary atm, especially since this data was already here

Reply:
Sure, we can do it later